etc/docker/node-init.sh

```diff
@@ -24,6 +24,29 @@
 
 cp -rf /var/mizar /home/
 mkdir -p /etc/cni/net.d
+# check if python is installed ,if yes and check version and install dev pkgs accordingly
+pyversion="3.7"
+
+version=$(nsenter -t 1 -m -u -n -i python3 -V 2>&1 | grep -Po '(?<=Python )(.+)')
+if [[ -z "$version" ]]
+then
+    echo "No Python!"
+    # continue with 3.7
+else
+    parsedVersion=$(echo "${version//./}")
+    if [[ "$parsedVersion" -gt "380" && "$parsedVersion" -lt "390" ]]
+    then
+       echo "Valid version 380"
+       pyversion="3.8"
+    elif [[ "$parsedVersion" -gt "370" && "$parsedVersion" -lt "380" ]]
+    then
+        echo "valid version 370"
+        #test purpose ,we can remove this elif block
+    fi
+fi
+
+echo "python"$pyversion
```

Comment:
My system has Python versioned 3.8 but it still gives following logs:
![image](https://user-images.githubusercontent.com/59909227/122364731-a585cf80-cf77-11eb-9141-007d2ee9ab18.png)

Python version I have in host machine is 3.8.5
![image](https://user-images.githubusercontent.com/59909227/122364805-b9313600-cf77-11eb-8221-c56ccf770341.png)

Reply:
What is output for 
nsenter -t 1 -m -u -n -i python3 -V 2>&1 | grep -Po '(?<=Python )(.+)'
on your system please.

Reply:
I double checked on ubuntu 20 (which has 3.8.5 ) and works clean.